RealtimeTwitterToHBase/README.rst

```diff
@@ -0,0 +1,59 @@
+Realtime Twitter To HBase Adapter Configuration
+===============================================
+
+Lets say we want to fetch Tweets from Twitter in Realtime and persist it in an HBase Table.
+
+ETL Realtime Template can be used to create an Adapter that reads from a Realtime Source and persists it to a Sink. In this example, we will read messages from Twitter in realtime and use a TableSink to write the Tweets to HBase.
+
+The config.json contains a sample Adapter configuration that you can use to accomplish the above task. We choose the ETL Realtime Application Template since we want to create a Realtime Adapter. 
+We set the Application Template name using the "template" property.
+
+We chose the Source to be Twitter Source and provide the OAuth credentials as properties to the Source. Since we want to persist the Tweets to HBase, we use a Table Sink. For the Table Sink, we choose the table name to be tweetTable (which will be created if the Table doesn't exist already) and we choose the row key to be id field of the Record emitted by the Twitter Source.
+
+For setting the Source name, we use the "source" property. The Twitter Source expects four properties (OAuth credentials for Twitter).
+
+- AccessToken
+- AccessTokenSecret
+- ConsumerKey
+- ConsumerSecret
+
+Similarly, for selecting the Sink, we use the "sink" property. The Table Sink expects two properties.
+
+- name : Name of the Table Dataset.
+- schema.row.field : The field which is used as the row key in the Table.
+
+Since we don't use any transforms in this Adapter, we leave that as an empty array.
+
+::
+
+You can create and start the Adapter by using the CDAP CLI (or the UI, for a more visual approach).
```

Comment:
Actually I don't think this guide will work well if the reader uses the UI, since they have to figure out how to configure an adapter in the UI that matches the configuration from config.json.